src/jobflow_remote/cli/utils.py

```diff
@@ -315,6 +315,11 @@ def wrapper(*args, **kwargs):
                 "The active project could not be determined and it is required to execute this command. Please "
                 "check the formatting of your YAML."
             )
+        except ProjectParsingError:
+            exit_with_error_msg(
+                "The active projects config file could not be parsed. Please check the formatting of your YAML. "
```

Comment:
If I am not wrong the code will arrive here even if the project name is misspelled in the `jfremote_project` variable (or with the `-p` option in the CLI). I think the error message could reflect a broader range of cases. Maybe it can use directly the error message coming from the error:
```python
except ProjectParsingError as exc:
    msg = f"{getattr(exc, 'message', str(exc))} You can use the command 'jf project list -w' to get the parsing errors."
```
It is fine to improve the message raised in ProjectParsingError, if you think it is not clear enough. Also note that yaml is not the only format allowed for the configuration file. Although I suppose is the most diffused, I would not specify the format in the message.